metrics/http_middleware.lua

```diff
@@ -73,6 +73,9 @@ end
 -- @return value from observable function
 function export.observe(collector, route, handler, ...)
     return collector:observe_latency(function(ok, result)
+        if ok ~= true then
+            log.error('http handler returned %q', result)
```

Comment:
You can test it with luatest capture.

I don't know if you have the access, but you can see example here: https://github.com/tarantool/megafon-customer-search-index-app/blob/525ca9d9528336a0568af76b43b467a237fc52a9/test/integration/bootstrap_test.lua#L470-L481